internal/store/evictionpool.go

```diff
@@ -30,6 +33,26 @@ func (a ByIdleTime) Less(i, j int) bool {
 	return GetIdleTime(a[i].lastAccessedAt) > GetIdleTime(a[j].lastAccessedAt)
 }
 
+func (a ByCounterAndIdleTime) Len() int {
+	return len(a)
+}
+
+func (a ByCounterAndIdleTime) Swap(i, j int) {
```

Comment:
Where is this used?

Reply:
I think it is for implementing Sort interface for **ByCounterAndIdleTime** and using it like this 
```
sort.Sort(ByCounterAndIdleTime(pq.pool))
```
below at line 68